.circleci/config.yml

```
@@ -69,7 +69,10 @@ jobs:
       - copy-gradle-properties
       - run:
           name: Build dependency diff report
-          command: ./tools/dependency-tree-diff/dependency-tree-diff.sh
+          command: |
+            if [ "$CIRCLE_PULL_REQUEST" ]; then
```

Prefer using `-n` for this kind of checks (just like Installable Builds step do)

```suggestion
            if [ -n "$CIRCLE_PULL_REQUEST" ]; then
```